internal/logger/logger.go

```diff
@@ -0,0 +1,96 @@
+package logger
+
+import (
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+var sugar zap.SugaredLogger
+var Log *zap.Logger = zap.NewNop()
```

Comment:
NEED CHANGES: please don't use the global variables. What if you use this logger in the several functions at the same time?